app/controllers/campuses_controller.rb

```diff
@@ -15,6 +16,7 @@ def new
   def create
     @campus = Campus.new(campus_params)
     save_for_html_json(@campus, "show") { campus_path(@campus) }
+    @campus.update(company_id: current_user.company.id)
```

Comment:
Never mind - changed this to just add the company id before save_for_html_json is called